app/controllers/formbuilder/edit.js

```diff
@@ -48,7 +50,7 @@ export default class FormbuilderEditController extends Controller {
     if (resetBuilder) {
       this.formChanged = true;
       this.builderStore.deregisterObserver();
-      this.builderStore = '';
+      this.builderStore = ForkingStoreHelper.getEmptyStoreValue();
```

Comment:
Nitpick / general feedback: I like the idea of renaming things more clearly so the code is readable, but I think it might be a bit overkill here. If I read the util name I expected that I would receive a new (empty) ForkingStore instance instead of an empty string, but maybe that's just me. 😄

I would probably just reset to `null` myself without a util.

Anyway, it seems that the "edit code flow" is broken at the moment though because of this line (also on master). Because of the async code happening after this, the page rerenders before a new ForkingStore instance is created and the form components receive an empty string instead of the instance which triggers an exception.

So this can probably be merged as-is and refactored in a separate PR when fixing the code editor flow.